README.md

```diff
@@ -134,7 +134,6 @@ To run the project, execute the following command:
 
```

Comment:
- [ ] @Timbar09 great work on your documentation, however, your issue link is not working in your URL is wrong instead of `https://github.com/Timbar09/Blog_App-Ruby/issues` you should have used `https://github.com/Timbar09/Blog_App-Rails/issues `

Reply:
Thanks for the feedback, @danmainah. I'll fix the issue right away.